pdf/pdfcanvas.cpp

```diff
@@ -398,6 +398,10 @@ QSGNode* PDFCanvas::updatePaintNode(QSGNode* node, QQuickItem::UpdatePaintNodeDa
             t = new QSGTransformNode{};
             root->appendChildNode(t);
         }
+        else if( t->childCount() > 0 ) {
+            delete t->firstChild();
+            t->removeAllChildNodes();
```

Comment:
This would delete and below recreate child nodes when view is scrolled even a bit. Should avoid if it's already the wanted type of node?